docs/sources/storage/_index.md

```diff
@@ -238,6 +238,64 @@ storage_config:
       dynamodb_url: dynamodb://region
 ```
 
+The role should have a policy with the following permissions attached.
```

Comment:
We should pronably factor everything in the storage section into `operations/storage` and `installation`.